src/context/directory/handlers/branding.ts

```diff
@@ -40,7 +40,10 @@ function parse(context: DirectoryContext): ParsedBranding {
     });
     definition.body = loadFileAndReplaceKeywords(
       path.join(brandingTemplatesFolder, definition.body),
-      context.mappings
+      {
+        mappings: context.mappings,
+        disableKeywordReplacement: context.disableKeywordReplacement,
+      }
```

Comment:
Most of the diffs in this PR are due to the changing function signature of `loadFileAndReplaceKeywords` which adds an option to disable keyword replacement.